lib/ansible/modules/windows/win_service.py

```diff
@@ -292,4 +318,18 @@
     returned: success and service exists
     type: list
     sample: false
+recovery:
```

Comment:
The recovery actions should also reflect the input way, i.e. returns a list of actions rather than a dict with `<x>_failure_<y>` and so forth. Basically it would just look similar to how you are defining `recovery_actions` as a module option right now.